main.go

```diff
@@ -50,43 +47,31 @@ func main() {
 	// Open a websocket connection to Discord and begin listening.
 	err = dg.Open()
 	if err != nil {
-		fmt.Println("error opening connection,", err)
-		return
+		log.Fatalln("error opening connection,", err)
 	}
 
 	// 5 AM everyday Monday - Friday
 	go func() {
 		c := cron.New()
-		c.AddFunc("0 5 * * 1-5", func() {
-			fmt.Println("test")
-			err := reminderRoutine(dg)
+		c.AddFunc("@midnight", func() {
```

Comment:
Can you set it to 5am or maybe an hour earlier?

I originally didn't set it to midnight because we're usually awake at midnight, but I want the message to be the first thing someone sees in the morning

Reply:
only proven to work using parsed intervals. regular crontab values not proven to work

2nd patch to reminders should solve this